geissonator/openbmc-events/openbmc-sfw

```diff
@@ -6,6 +6,9 @@ import tarfile
 import time
 import json
 
+import urllib3
+urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
```

Comment:
We should let the user choose. See what I did for `openbmc-sfw` (which I pushed without your review accidentally).

Reply:
Done

Reply:
Well, I did this in openbmc-events at least.  Looks like you already did openbmc-sfw. Guess I need to get this commit stream rebased.